Makefile.am

```diff
@@ -457,7 +457,9 @@ libgeopmpolicy_la_SOURCES = src/Accumulator.cpp \
                             src/record.hpp \
                             # end
 
-beta_source_files = src/Daemon.cpp \
+beta_source_files = src/CPUActivityAgent.cpp \
```

Comment:
We should probably add a [.. note::](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html) block to the .rst files for these files that specifies they are behind the beta flag.